wgsl/index.bs

```diff
@@ -2683,8 +2682,8 @@ is terminated.
 Otherwise, evaluation continues with the next expression or statement after
 the evaluation of the call site of the current function invocation.
 
-If the return type of the function is the void type, then the return statement
-must not have an expression.
+If the return type of the function is the void type, then the return statement is
+optional. If the return statement is provided for a void function it must not have an expression.
```

Comment:
The statement about not having an expression is already covered in the description of the return statement itself.
https://gpuweb.github.io/gpuweb/wgsl.html#return-statement

Reply:
I'm confused, this text is in the return-statement block?

Reply:
The rule has already been captured in the spec section for the "Return statement".

Reply:
Generally it's a spec anti-pattern to duplicate the same rule in multiple places.  Maybe this doesn't matter too much.